include/base/sparsity_pattern.h

```diff
@@ -107,6 +107,20 @@ class Build : public ParallelObject
          const bool implicit_neighbor_dofs_in,
          const bool need_full_sparsity_pattern_in);
 
+  /**
+   * Special functions.
+   * - The Build object holds references to DofMap and coupling
+   *   functors, therefore it can't be move-assigned.
+   */
+  Build (const Build &) = default;
+  Build & operator= (const Build &) = default;
```

Comment:
You can't default copy-assign objects with reference members either, as that would be re-seating a reference. Copy-construct should be OK though. Some compilers correctly warn about this, but others don't.